src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -64,6 +69,12 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
         $this->urlGenerator = $urlGenerator;
         $this->subresourceOperationFactory = $subresourceOperationFactory;
         $this->nameConverter = $nameConverter;
+        if (!$serializerContextBuilder) {
+            @trigger_error(sprintf('Not passing an instance of %s to %s() is deprecated since version 2.5 and will be removed in 3.0.', SerializerContextBuilderInterface::class, __METHOD__), E_USER_DEPRECATED);
```

Comment:
Is this deprecation mandatory? Havin as few mandatory deps as possible is always better, it allows to reuse the class easily when installing API Platform as a standalone library.